tests/resources/SenderResource.test.ts

```diff
@@ -0,0 +1,249 @@
+              "accept": "application/json",
+              "authorization": "Bearer mockedToken",
+              "content-type": "application/json",
+            },
+            "method": "GET",
+          },
+        ]
+      `);
+    });
+  });
+
+  describe('getOne', () => {
+    it('sends the request to the correct URL', async () => {
+      await senders.getOne('sender1');
+
+      expect(mockedFetch.mock.calls[0]).toMatchInlineSnapshot(`
+        Array [
+          "http://demo.com/api/account/senders/sender1",
+          Object {
+            "headers": Object {
+              "accept": "application/json",
+              "authorization": "Bearer mockedToken",
+              "content-type": "application/json",
+            },
+            "method": "GET",
+          },
+        ]
+      `);
+    });
+  });
+
+  describe('delete', () => {
+    it('sends the request to the correct URL', async () => {
+      await senders.delete('sender1');
+
+      expect(mockedFetch.mock.calls[0]).toMatchInlineSnapshot(`
+        Array [
+          "http://demo.com/api/account/senders/sender1",
+          Object {
+            "headers": Object {
+              "accept": "application/json",
+              "authorization": "Bearer mockedToken",
+              "content-type": "application/json",
+            },
+            "method": "DELETE",
+          },
+        ]
+      `);
+    });
+  });
+
+  describe('invite', () => {
+    it('sends the request to the correct URL', async () => {
+      await senders.invite('sender1');
+
+      expect(mockedFetch.mock.calls[0]).toMatchInlineSnapshot(`
+        Array [
+          "http://demo.com/api/account/senders/sender1/invite",
+          Object {
+            "headers": Object {
+              "accept": "application/json",
+              "authorization": "Bearer mockedToken",
+              "content-type": "application/json",
+            },
+            "method": "POST",
+          },
+        ]
+      `);
+    });
+  });
+
+  describe('getDefaultSignature', () => {
+    it('sends the request to the correct URL', async () => {
+      mockedFetch.mockResolvedValue({
+        ok: true,
+        json() {
+          return Promise.resolve({
+            fileName: 'file-name',
+            mediaType: 'image/png',
+            content: Buffer.from('abc1234').toString('base64'),
+          });
+        },
+      } as Response);
+
+      const response = await senders.getDefaultSignature('sender1');
+
+      expect(mockedFetch.mock.calls[0]).toMatchInlineSnapshot(`
+        Array [
+          "http://demo.com/api/account/senders/sender1/signature/image",
+          Object {
+            "headers": Object {
+              "accept": "application/json",
+              "authorization": "Bearer mockedToken",
+              "content-type": "application/json",
+            },
+            "method": "GET",
+          },
+        ]
+      `);
+
+      expect(response.fileName).toStrictEqual('file-name');
```

Comment:
```suggestion
      expect(response.fileName).stringContaining('file-name');
```

I think `toStrictEqual` is better for cases where we need to compare data structures like objects and array.

Reply:
Do you mean `stringMatching`?

Reply:
stringContaining could return true on the string "I-l0ve-file-name-stuffs"

Reply:
Also, the proper usage should be
```
expect(response.fileName).toBe(expect.stringMatching('file-name'));
```

Seems like it's the exact same as `toStrictEqual`

Reply:
Original looks better (legibility wise) than that nested expect version